internal/eval/commands.go

```diff
@@ -96,6 +96,14 @@ var (
 		NewEval:    evalGET,
 	}
 
+	randomKeyCmdMeta = DiceCmdMeta{
+		Name:       "RANDOMKEY",
+		Info:       `RANDOMKEY returns a random key from the currently selected database.`,
+		Arity:      1,
+		IsMigrated: true,
```

Comment:
IsMigrated should not be set to true since this command does not support multi-threaded operation yet.

Reply:
Ok, then i guess i will have to revert things to use the old eval method here